sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -52,3 +52,36 @@ export interface ProgramOfferingDto extends ProgramOfferingBaseDTO {
 export interface ProgramOfferingDetailsDto {
   studyStartDate?: Date;
 }
+
+/**
+ * Transformation util for Program Offering.
+ * @param offering
+ * @returns ProgramOfferingDto
+ */
+export const transformToProgramOfferingDto = (
+  offering: ProgramOfferingModel,
+): ProgramOfferingDto => {
+  return {
+    id: offering.id,
+    offeringName: offering.name,
+    studyStartDate: offering.studyStartDate,
+    studyEndDate: offering.studyEndDate,
+    actualTuitionCosts: offering.actualTuitionCosts,
+    programRelatedCosts: offering.programRelatedCosts,
+    mandatoryFees: offering.mandatoryFees,
+    exceptionalExpenses: offering.exceptionalExpenses,
+    tuitionRemittanceRequestedAmount: offering.tuitionRemittanceRequestedAmount,
+    offeringDelivered: offering.offeringDelivered,
+    lacksStudyDates: offering.lacksStudyDates,
+    lacksStudyBreaks: offering.lacksStudyBreaks,
```

Comment:
is lacksStudyBreaks a mandatory field that will always have a value returned ? If not please can you add a check for null ?

Reply:
I had a look into what is discussed yesterday, we are actually checking if `offerings` is there or not before the transforming to dto (`line 360`),  so, we need not need an offering null, 
![image](https://user-images.githubusercontent.com/77353155/152406972-074f58e9-9fd7-4330-8e99-dffcd2e6ef16.png)